src/CMakeLists.txt

```diff
@@ -414,17 +414,84 @@ if (WITH_DOTNET_WRAPPING)
 	include(${FESAPI_ROOT_DIR}/cs/CMakeLists.txt)
 endif (WITH_DOTNET_WRAPPING)
 
+
 # ============================================================================
-# Install FesapiCpp library
+# Create package
 # ============================================================================
+include(CMakePackageConfigHelpers)
 
-INSTALL (
-	TARGETS ${CPP_LIBRARY_NAME}
-	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+######## 1. Build tree
+
+export(TARGETS ${CPP_LIBRARY_NAME}
+	NAMESPACE Fesapi::
+	FILE "${CMAKE_BINARY_DIR}/lib/cmake/${CPP_LIBRARY_NAME}/${CPP_LIBRARY_NAME}Targets.cmake"
+)
+
+export(PACKAGE ${CPP_LIBRARY_NAME})
+
+configure_package_config_file(
+	"${CMAKE_CURRENT_SOURCE_DIR}/${CPP_LIBRARY_NAME}Config.cmake.in"
```

Comment:
This file should go in cmake directory not in src directory except if you think this is a bad idea.
Indeed, for now, all files tweaked/configured by cmake are in this cmake directory.

Reply:
Ok, I have the habits to put it near the code that is packaged in a library. Here, there is only one library so yes I will change it to the cmake directory